docs/README.md

```diff
@@ -1466,52 +1468,15 @@ The above webhook URL is a mandatory parameter. Make sure to [Create Incoming We
 
 Sample of a SUCCESS notification on a `push` event.
 
-![Teams Success](./images/teams-success.png)
-
-Sample of a FAILURE notification on a `push` event.
+![Teams Success](./images/send-teams-push-success.png)
```

Comment:
The docs here mentioned creating the hook via the "[Incoming Webhook](https://learn.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?tabs=newteams%2Cdotnet)" approach that is going to be deprecated by MSFT soon.

I was curious whether we tried at all the [new approach based on Workflows](https://support.microsoft.com/en-us/office/create-incoming-webhooks-with-workflows-for-microsoft-teams-8ae491c7-0394-4861-ba59-055e33f75498) after the changes? It'd be great to avoid having to rework this in the near future 😅